sci-libs/scikit-learn/files/scikit-learn-0.24.0-no-O3.patch

```diff
@@ -0,0 +1,55 @@
+diff --git a/sklearn/manifold/setup.py b/sklearn/manifold/setup.py
+index 0db2d5d..fb4fc38 100644
+--- a/sklearn/manifold/setup.py
++++ b/sklearn/manifold/setup.py
+@@ -15,14 +15,14 @@ def configuration(parent_package="", top_path=None):
+     config.add_extension("_utils",
+                          sources=["_utils.pyx"],
+                          include_dirs=[numpy.get_include()],
+-                         libraries=libraries,
+-                         extra_compile_args=["-O3"])
```

Comment:
can you talk to upstream abotu this?

Reply:
I'll definitely ask them but...
honestly, this build system is so bad, it keeps changing every version.
like in last version they weren't even using these functions. I've like no clue what they are gonna do next version.
so frigging irritating.